Source/ACE.Server/WorldObjects/WorldObject_Use.cs

```diff
@@ -60,21 +60,22 @@ public virtual void OnActivate(WorldObject activator)
         {
             //Console.WriteLine($"{Name}.OnActivate({activator.Name})");
 
-            // when players double click an object in the 3d world,
-            // and the packet comes in as GameAction 0x35 - UseWithTarget
+            // when players double click an object,
+            // and the packet comes in as GameAction 0x36 - UseItem
             // from the game perspective, technically this starts as an 'Activate',
             // which can have a list of possible ActivationResponses - 
             // Use (by far the most common), Animate, Talk, Emote, CastSpell, Generate
 
-            // PropertyInt.Active indicates if this object can be activated, default is true
-            if (!Active) return;
+            // PropertyInt.Active indicates if this object can be activated, default is 1
+            if (Active == 0) return;
```

Comment:
this should be an enum, instead of having magic values (0, 1, 0xFFFF)

Reply:
See above